README.md

```diff
@@ -8,8 +8,15 @@ docker-compose exec phpfpm composer install
 docker-compose exec phpfpm bin/console doctrine:migrations:migrate --no-interaction
 ```
 
+Open the site:
+
+```sh
+open "http://$(docker-compose port nginx 80)"
```

Comment:
do we not open in nginx port 8080

Reply:
No, `nginx` is running on port `80` (cf. https://github.com/itk-dev/mtm-radar/blob/feature/hep-hey/docker-compose.yml#L53).